content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -627,13 +631,55 @@ read-modify-write and/or patch are the following:
 It is strongly recommended for controllers to always "force" conflicts, since they
 might not be able to resolve or act on these conflicts.
 
+### Transferring Ownership
+
+In addition to the concurrency controls provided by [conflict
+resolution](#conflicts), Server Side Apply provides ways to perform coordinated
+field ownership transfers from users to controllers:
+
+First the user signals to the controller that it should take ownership of a
+field, usually by setting some other field (perhaps in some other object) that the controller observes. The
+controller then claims ownership of the field by updating the object or applying a configuration
+containing the field. This can be done in one of the following ways:
+
+- If the controller is designed to immediately set the value of the field, the
+controller should apply with force set to true. Once the user observes that the
+controller has taken ownership of the field, they can remove the field from
+their applied configuration to complete the transfer.
+- If the controller will not set the value of the field immediately, the user
+should genereate a unique, temporary, field manager name, and transfer ownership
+of the field to it. This transfer should be done by first setting the field to
+the same value in the applied configrations of the user and temporary field
+manager, and then by removing it from the user's applied configuration after
+field ownership is shared. Later, when the controller does want to set the value
+of the field, it can then apply with force set to true, which will automatically
+delete the temporary field manager without the need for any clean up.
+
```

Comment:
I hesitate to ask for more work, but I feel like a concrete example of handing off a deployment's replica count to an HPA would make this easier to understand for people?

Reply:
no problem, I'll add it.

Reply:
Restructured this section a bit to try to make the example flow: https://deploy-preview-22412--kubernetes-io-vnext-staging.netlify.app/docs/reference/using-api/api-concepts/#transferring-ownership

I'm testing the example on an apiserver now to make sure it actually works :)

Reply:
Works after fixing the example deployment file. Added a new example directory for server side apply so I can have example yaml that is not shared with other parts of the docs.